### ISSUE TITLE
ref is depecrated

### ISSUE THREAD

**Issue by shai32** [attently/riek#3]
RIEStatefulBase use ref attribute is a string, and it's deprecated 
https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs

we should fix this.

also getting errors 
![image](https://user-images.githubusercontent.com/16700996/41995234-aaedc872-7a59-11e8-8863-a179a8788215.png)
I think its related to the string ref

**Comment by shai32** [comment]
if I will do a pr for this. is there anybody that can merge it?

**Comment by FizzyGalacticus** [comment]
Yes! My apologies for not getting back sooner. If you have a solution ready, definitely submit a PR!

**Comment by shai32** [comment]
ok, pr added, can you review?

**Comment by shai32** [comment]
?

**Comment by dyoung522** [comment]
I guess this project is dead as well.

**Comment by FizzyGalacticus** [comment]
My apologies, I am no longer with @attently and therefore have not been overseeing this project. I'm currently in talks with them to verify that I can still be a part of this project under the Attently GitHub.

I have thought about `riek` several times in the past few weeks, and I believe that in order to uphold maintainability, there are quite a few things in the underlying structure of this project that would need to be fixed. If I can find some time in the near future, I'd be interested in tearing `riek` apart and starting from the ground up with more compartmentalized implementations.
